### PR TITLE
3.2.1 API

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: Ds3/Calls/CancelActiveJobSpectraS3Request.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{
+    public class CancelActiveJobSpectraS3Request : Ds3Request
+    {
+        
+        public string ActiveJobId { get; private set; }
+
+        
+
+        
+        public CancelActiveJobSpectraS3Request(Guid activeJobId) {
```
Comment (shabtaisharon) [inline Ds3/Calls/CancelActiveJobSpectraS3Request.cs]: your { should be in a new line

File: Ds3/Calls/CancelActiveJobSpectraS3Request.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+using System;
+using System.Net;
+
+namespace Ds3.Calls
+{
+    public class CancelActiveJobSpectraS3Request : Ds3Request
+    {
+        
+        public string ActiveJobId { get; private set; }
+
```
Comment (shabtaisharon) [inline Ds3/Calls/CancelActiveJobSpectraS3Request.cs]: why so many new lines?

Comment (RachelTucker) [inline Ds3/Calls/CancelActiveJobSpectraS3Request.cs]: consequence of using the templates

File: Ds3/Calls/CancelActiveJobSpectraS3Request.cs (hunk 1 above)
Comment (RachelTucker) [inline Ds3/Calls/CancelActiveJobSpectraS3Request.cs]: was this not an issue in 3.0 sdk?

Comment (shabtaisharon) [inline Ds3/Calls/CancelActiveJobSpectraS3Request.cs]: i dont think i reviewed that, so i cant really say

File: Ds3/Calls/CancelAllActiveJobsSpectraS3Request.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        public JobRequestType? RequestType
+        {
+            get { return _requestType; }
+            set { WithRequestType(value); }
+        }
+
+        public CancelAllActiveJobsSpectraS3Request WithBucketId(Guid? bucketId)
+        {
+            this._bucketId = bucketId.ToString();
+            if (bucketId != null) {
```
Comment (shabtaisharon) [inline Ds3/Calls/CancelAllActiveJobsSpectraS3Request.cs]: same { issue

**Pull request merged**